src/CodeEditor/driver.js

```diff
@@ -132,4 +132,23 @@ export default class CodeEditorDriver
         this.wrapper.simulate( 'mouseleave' );
         return this;
     }
+
+    pressKey( keyCode )
+    {
+        const props = this.wrapper.props();
+        const onKeyPress = this.wrapper.prop( 'onKeyPress' );
+
+        if ( props.isDisabled )
+        {
+            throw new Error( ERR
+                .CODEEDITOR_ERR( 'pressKey', 'disabled' ) );
+        }
+
+        if ( onKeyPress )
+        {
+            onKeyPress( keyCode );
```

Comment:
Hmmm  looking at the `handleKeyPress` implementation in `CodeEditor/index.jsx` we don’t actually call the `onKeyPress` callback with any arguments (no `keyCode`). This means that this driver method would not provide an accurate simulation of the component’s behavior.

So either this line changes to:
```suggestion
            onKeyPress();
```

or we need to change the `handleKeyPress` implementation in `CodeEditor/index.jsx`.